notebooks/233-blip-visual-language-processing/233-blip-quantize-nncf.ipynb

```diff
@@ -0,0 +1,724 @@
+{
```

Comment:
<p>tatle prepare dataset, but you initialize model instead.</p><p>Also I'm doubt regarding your dataset choice,TextVQA is a dataset to benchmark visual reasoning based on text in images, but BLIP model trained analyze image content, not read text represented on it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
@eaidova do you have any suggestions for a more appropriate dataset?

Reply:
if you prefer to use hf datasets, I'll recomend visual genome dataset (it contains annotation for vqa task)
https://huggingface.co/datasets/visual_genome

generally, the most popular for VQA is VQA v2 dataset https://visualqa.org/, but unfortunatly in hub I see only translated on vietnameese its version...

Reply:
visual genome dataset is too small for quantization. QA v1.2 subset contains 13 unique images. I found https://huggingface.co/datasets/HuggingFaceM4/VQAv2 dataset, looks like this is the HF version of https://visualqa.org/.